src/constants/cache.ts

```diff
@@ -1,3 +1,6 @@
-export const CACHE_NAME = "CPR_search_cache";
+// const week = 1000 * 60 * 60 * 24 * 7;
```

Comment:
Should they stay or should they go? :notes:

Reply:
GO 🔥